cli.js

```diff
@@ -92,33 +95,33 @@ st.once('testserver', function (server) {
 
 	setState('download');
 	var ping = Math.round(server.bestPing);
-	stats.ping = (cli.flags.json) ? ping : chalk.cyan(ping + chalk.dim(' ms'));
+	stats.ping = cli.flags.json ? ping : chalk.cyan(ping + chalk.dim(' ms'));
 });
 
 st.on('downloadspeedprogress', function (speed) {
 	if (state === 'download' && cli.flags.json !== true) {
-		var download = roundTo(speed, speed > 10 ? 0 : 1);
-		stats.download = chalk.yellow(download + chalk.dim(' Mbps'));
+		var download = roundTo(speed * multiplier, speed > 10 ? 0 : 1);
```

Comment:
@SamVerschueren Seems like something wrong in code, will investigate and update again.
 --bytes is not working now.

Results:
SrinivasanSekar:speed-test srinivasansekar$ speed-test

      Ping   113 ms
  Download   2.7 Mbps
    Upload   2.1 Mbps
SrinivasanSekar:speed-test srinivasansekar$ speed-test --bytes

      Ping   98 ms
  Download   1.8 Mbps
    Upload   2.2 Mbps

I think ternaries are not working.